src/App.js

```diff
@@ -44,6 +45,39 @@ class App extends Component {
 
     this.setState({ habits: newHabits })
   }
+
+  increaseCount(id) {
+    const allHabits = this.state.habits
+    const habitIndex = allHabits.findIndex(habit => habit.id === id)
+    const oldHabit = allHabits[habitIndex]
+
+    const newHabits = [
+      ...allHabits.slice(0, habitIndex),
+      { ...oldHabit, count: oldHabit.count + 1 },
+      ...allHabits.slice(habitIndex + 1)
+    ]
+
+    this.setState({ habits: newHabits })
+  }
+
+  decreaseCount(id) {
```

Comment:
Diese Funktion ist fast gleich mit der anderen. Kannst du abstrahieren, wie in meinem Beispielprojekt.